manifests/vcsscm/git.pp

```diff
@@ -20,6 +22,26 @@
           ensure => $ensure,
         }
       }
+
+      if $bash_completion {
+        package { 'bash-completion':
+          ensure => $ensure,
+        }
+      }
+
+      if $bash_prompt {
+        vcsrepo { '/opt/bash-git-prompt/':
```

Comment:
Can you add the vcsrepo as a dependency here:
https://github.com/edestecd/puppet-software/blob/master/metadata.json#L12

Everything else looks good. After that I will merge.